config.go

```diff
@@ -41,6 +41,20 @@ const (
 	defaultNumChanConfs       = 3
 	defaultNoEncryptWallet    = false
 	defaultTrickleDelay       = 30 * 1000
+
+	// minTimeLockDelta is the minimum timelock we require for incoming
+	// HTLCs on our channels.
+	minTimeLockDelta = 4
+
+	defaultBitcoinMinHTLCMSat   = 1000
```

Comment:
There're already the `defaultXXXForwardingPolicy` structs in `chainregistry.go`. With these definitions it's all duplicated again. When setting the defaults on the config to be parsed, those structs can be used directly as they're statically initialized.

Reply:
I removed the structs from `chainregistry`. Do you prefer we keep the constants there?